crates/fs/src/fs.rs

```diff
@@ -995,7 +1001,7 @@ impl FakeFs {
         let new_mtime = state.next_mtime;
         let new_inode = state.next_inode;
         state.next_inode += 1;
-        state.next_mtime += Duration::from_nanos(1);
+        state.next_mtime += Duration::from_nanos(100);
```

Comment:
this is because: on Windows the time is represented in 100 nanosecond intervals

Reply:
Let's move away this 100 into an internal constant and comment that, ideally with the link to the docs.